Remotion/ObjectBinding/Web/Res/Themes/NovaViso/HTML/BocList.css

```diff
@@ -1,594 +1,340 @@
-th.bocListTitleCell > a[href]:hover
+div.bocList table.bocListTable td.bocListDataCellIndex
 {
-  color: Black;
-  padding: 1px;
-  border: 1px solid #7CB2E6;
-  background-color: #E5F0FC;
-  outline: none;
+  text-align: right;
 }
+/*endregion*/
 
-th.bocListTitleCell > a[href]:active
+/*region bocListTableBlock*/
+div.bocList div.bocListTableBlock
 {
-  color: Black;
-  border: solid 1px #569CE0;
-  background-color: #C5E0FC;
+  grid-column: 1 / span 2;
+  grid-row: 1;
+  display: grid;
+  grid-template-rows: auto min-content;
+  height: 100%;
 }
 
-th.bocListTitleCell .bocListSortingOrder img
+div.bocList div.bocListTableBlock.hasMenuBlock
 {
-  height: 16px;
-  width: 16px;
+  grid-column: 1;
+  border-right: 1px solid var(--boclist-tablecell-border-color);
 }
+/*endregion*/
 
-th.bocListTitleCell > a[href]:focus .bocListSortingOrder img,
-th.bocListTitleCell > a[href]:hover .bocListSortingOrder img,
-th.bocListTitleCell > a[href]:active .bocListSortingOrder img
+/*region bocListNavigator*/
+div.bocList div.bocListTableBlock div.bocListNavigator
 {
-  -webkit-filter: brightness(0);
-  filter: brightness(0);
-  filter: progid:DXImageTransform.Microsoft.BasicImage(grayscale=1);
+  overflow: hidden;
+  padding: var(--remotion-themed-spacing-half);
+  display: flex;
+  align-items: center;
+  grid-row: 2;
 }
 
-/*  Style applied to the cells in the BocList's odd and even data rows. */
-td.bocListDataCell
+div.bocList div.bocListTableBlock div.bocListNavigator > span
 {
-  text-align: left;
-
-  padding: 2px 5px 2px 2px;
-  border-top: none;
-  border-right: none;
-  border-left: none;
-  border-bottom: 1px solid #C8C8C8;
-
-  cursor: default;
+  margin-right: var(--remotion-themed-spacing);
 }
 
-/*  Style applied to the cells in the BocList's not-selected odd and even data rows. */
-tr.bocListDataRow.odd td.bocListDataCell
+div.bocList div.bocListTableBlock div.bocListNavigator > span > input
 {
-  background-color: White;
+  min-height: unset;
+  height: var(--remotion-themed-line-height);
+  padding: var(--remotion-themed-spacing-half);
+  margin: var(--remotion-themed-spacing-half) var(--remotion-themed-spacing);
+  text-align: right;
+  width: calc(0.6em * var(--size));
 }
 
-tr.bocListDataRow.even td.bocListDataCell
+div.bocList div.bocListTableBlock div.bocListNavigator img
 {
-  background-color: #F0F0F0;
+  height: var(--remotion-themed-line-height);
+  width: var(--remotion-themed-line-height);
+  border: var(--remotion-themed-border-width) solid transparent;
+  border-radius: 2px;
+  padding: var(--remotion-themed-spacing-half);
+  margin: 0 var(--remotion-themed-spacing-half);
+  background-color: var(--remotion-themed-control-background);
 }
 
-/*  Style applied to the cells in the BocList's selected odd and even data rows. */
-tr.bocListDataRow.selected td.bocListDataCell
+div.bocList div.bocListTableBlock div.bocListNavigator a[href] img
 {
-  background-color: #C5E0FC;
+  border-color: var(--color-standard-border);
```

Comment:
just realized this, this border is on the img, not the anchor. that's not optimal. the border and the anchor should have the same size, probably best to just move the border to the anchor.